File: Python ejs/UD03/UD03A031.py
n1=int(input("Introduzca un número: "))
n2=int(input("Introduzca un número: "))
n3=int(input("Introduzca un número: "))

if n1==n2 and n2==n3:
    print("Los tres son iguales")
elif n1!=n2 and n1!=n3 and n2!=n3:
    print("Los tres son distintos")
else:
    print("Dos números son iguales")

File: Java ejs/UD3/A03/UD03A031.java
import java.util.Scanner;
public class UD03A031{
	public static void main(String[] args){
		int n1,n2,n3;
        Scanner teclado=new Scanner(System.in);
        System.out.println("Introduzca un número: ");
        n1=teclado.nextInt();
        System.out.println("Introduzca un número: ");
        n2=teclado.nextInt();
        System.out.println("Introduzca un número: ");
        n3=teclado.nextInt();
        if (n1==n2 && n2==n3)
        {
            System.out.println("Los tres son iguales");
        }else{
            if (n1!=n2 && n1!=n3 && n2!=n3)
            {
                System.out.println("Los tres son distintos");
            }else{
                if ( n1==n2 || n1==n3 || n2==n3)
                {
                    System.out.println("Dos números son iguales");
                }
            }
        }
	}
}